ceph/rados/pool_workflows.py

```diff
@@ -163,21 +163,29 @@ def prepare_static_data(self, node):
         return tmp_file
 
     def do_rados_put(
-        self, client, pool: str, obj_name: str = None, nobj: int = 1, offset: int = 0
+        self,
+        client,
+        pool: str,
+        obj_name: str = None,
+        nobj: int = 1,
+        offset: int = 0,
+        timeout: int = 600,
     ):
         """
         write static data to one object or nobjs in an app pool
         Args:
             client: client node
             pool: pool name to which data needs to added
             obj_name (optional): Name of the existing object or
-            new object to be created in the pool
+                                 new object to be created in the pool
             nobj: Number of times data will be put to object 'obj_name'
-            with incremental offset if 'obj_name' is specified, else
-            data will be put once to nobj number of objects -
-            obj0, obj1, obj2 and so on...
+                  with incremental offset if 'obj_name' is specified, else
+                  data will be put once to nobj number of objects -
+                  obj0, obj1, obj2 and so on...
             offset: write object with start offset, default - 0
```

Comment:
I believe even this (nobj, offset ) is optional.

Can you also add a example for method invocation in docstring?

Reply:
Sure, added with latest commit